dev/devicelab/manifest.yaml

```diff
@@ -373,6 +373,7 @@ tasks:
       Tests the `flutter attach` command.
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
+    timeout_in_minutes: 20
```

Comment:
can we make the test faster? having a test take more than 5 minutes, let alone more than 15 (the default timeout), is really long. We only have a small amount of time to run every test and not get behind. Is this test really worth so much that we want to pay 20 minutes for it?

Reply:
Multipart answer:

- If we had a choice between not having this test paying 20 min, is it worth it? Yes
- Can we make the totality of the test faster? Unlikely. The slowest part would be building the APK during the setup. Splitting the test isn't very efficient since the attaching(-under-test) part of the validation : build the APK ratio is small. But 15 min works since many other devicelab tests also build the APK. I didn't realize the default was 15. 
- Can we make the test fail faster? Yes. That was the semi-philosophical question I was asking. Having a surgical timeout during the attach rather that a timeout for the entire test would make the test faster in cases of failures.

Reply:
Can you point me to the 15 min default? I couldn't find it. Seems like downstream from the manifest in https://github.com/flutter/flutter/blob/master/dev/devicelab/lib/framework/framework.dart#L89 we're just letting it run without timeout.